kytos/utils/metadata.py

```diff
@@ -2,7 +2,7 @@
 
 The present metadata is intended to be used mainly on the setup.
 """
-__version__ = '2021.1'
+__version__ = '2022.1b1'
```

Comment:
I'm not sure if the team has agreed on which semantic versioning we'll continue with, but I ended up using using the same one in place cc'ing @italovalcy @ajoaoff @rmotitsuki let me know if you have other suggestions or if this has already been specified somewhere else.

Reply:
Yeah, using the same approach sounds good to me!

Reply:
@italovalcy thanks for reviewing